pkgs/development/tools/misc/replacement/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib, buildPythonApplication, isPy3k
+, ruamel_yaml
+, pytest, sh
+}:
+
+buildPythonApplication rec {
+  pname = "replacement";
+  version = "0.4.4";
+
+  disabled = !isPy3k;
+
+  src = builtins.fetchGit {
+    url = "https://siriobalmelli@github.com/siriobalmelli/${pname}.git";
+    rev = "1b85abf70b27bb6f05325a5667cdce5276c2cf69";  # tagged v0.4.4
+  };
+
+  propagatedBuildInputs = [
+    ruamel_yaml
+  ];
+
+  checkInputs = [
+    pytest
+    sh
+  ];
+
+  checkPhase = ''
+    pytest
+  '';
```

Comment:
Try to put `pytestCheckHook` in `checkInputs` instead of this and `pytest`.

Reply:
That works nicely, thank you :)